consul_impl.go

```diff
@@ -39,7 +39,12 @@ type consulDataStoreImpl struct {
 }
 
 func newConsulDataStoreImpl(builder *DataStoreBuilder, loggers ldlog.Loggers) (*consulDataStoreImpl, error) {
-	loggers.Infof("Using config: %+v", builder.consulConfig)
+	loggers.SetPrefix("ConsulDataStore:")
+	addr := builder.consulConfig.Address
+	if addr == "" {
+		addr = c.DefaultConfig().Address
+	}
+	loggers.Infof("Using Consul server at %s", addr)
```

Comment:
Our current Consul integrations for most of the SDKs don't log anything at startup, but I think that was just an oversight - we do the equivalent for Redis and I think it's reasonable.